ibis/backends/__init__.py

```diff
@@ -1018,7 +1025,7 @@ def tables(self):
         >>> people = con.tables.people  # access via attribute
 
         """
-        return TablesAccessor(self)
+        return TablesAccessor(weakref.proxy(self))
```

Comment:
FWIW removing the `cached_property` here also seems sane (and simpler). Creating a `TablesAccessor` is super cheap, no reason to hold onto a reference to it on the backend. If that'd work, I'd prefer that to using weakrefs here at all.

Reply:
Yep, agree. Let's leave out the `weakref`s until there's a more concrete problem they solve.